File: sdkwork-backend-sdk-typescript/src/types/plus-ai-agent-query-list-form.ts
export interface PlusAiAgentQueryListForm {
  id?: number;
  createdAtFrom?: string;
  createdAtTo?: string;
  userId?: number;
  keyword?: string;
  status?: 'DEFAULT' | 'ACTIVE' | 'INACTIVE' | 'DELETED' | 'DRAFT';
  dataScope?: 'DEFAULT' | 'PRIVATE' | 'SHARED' | 'PUBLIC';
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusAiAgentQueryListForm.kt
package com.sdkwork.backend

data class PlusAiAgentQueryListForm(
    val id: Int? = null,
    val createdAtFrom: String? = null,
    val createdAtTo: String? = null,
    val userId: Int? = null,
    val keyword: String? = null,
    val status: String? = null,
    val dataScope: String? = null
)
